apollo-compiler/src/main/kotlin/com/apollographql/apollo3/compiler/codegen/kotlin/file/TestBuildersBuilder.kt

```diff
@@ -64,6 +63,8 @@ class TestBuildersBuilder(
 
   private fun typeSpec(): TypeSpec {
     return TypeSpec.objectBuilder(simpleName)
+        // Note: the compiler forbids referencing ApolloExperimental::class directly, except in an @OptIn context
+        .addAnnotation(ClassName("com.apollographql.apollo3.api", "ApolloExperimental"))
```

Comment:
👍  I also have the project to relocate `apollo-api` in the Gradle plugin jar too so that'll help.

Nit: there's a `ClassNames` file that tries to list all hardcoded class names like this, would be nice to move it there: https://github.com/apollographql/apollo-android/blob/d852dd92ee084732d49ae993f4e1346384f2fde7/apollo-compiler/src/main/kotlin/com/apollographql/apollo3/compiler/codegen/ClassNames.kt#L8

Reply:
Move it (and removed the comment which was no longer really useful)